.github/workflows/build_and_test.yml

```diff
@@ -217,6 +217,16 @@ jobs:
       run: |
         python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner plotly>=4.8
         python3.6 -m pip list
+    # TODO(SPARK-35507) Move Python 3.9 installtation to the docker image
+    - name: Install Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+        architecture: x64
```

Comment:
QQ: is it necessary to specify the architecture here?

Reply:
Seems not .. but let me just leave it for consistency with other places above, and just to be explicit.